src/log.cpp

```diff
@@ -139,8 +142,12 @@ namespace rhost {
                 stackdump_filename = log_dir / (filename + ".stack.dmp");
                 fulldump_filename = log_dir / (filename + ".full.dmp");
             }
-        
+
+#ifdef _WIN32
             logfile = _fsopen(log_filename.make_preferred().string().c_str(), "wc", _SH_DENYWR);
+#else
+            logfile = fopen(log_filename.make_preferred().string().c_str(), "wc");
```

Comment:
"c" is a VC-specific open mode (it disables buffering). It won't be supported on Linux. So this should be just "w".